securedrop/source_templates/index.html

```diff
@@ -12,7 +12,8 @@
     {% endassets %}
   </head>
   <body>
-    <div class="warning"><strong>We recommend turning the Security Slider to High to protect your anonymity:</strong> <a id="disable-js" href="/howto-disable-js">Learn how to set it to high</a>, or ignore this warning to continue. <img id="warning-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
+    <div class="js-warning warning"><strong>We recommend turning the Security Slider to High to protect your anonymity:</strong> <a id="disable-js" href="">Learn how to set it to high</a>, or ignore this warning to continue. <img id="js-warning-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
+    <div class="tor-warning warning"><strong>We strongly recommend using Tor Browser to access SecureDrop:</strong> <a id="disable-js" href="{{ url_for('not_using_tor_browser_warning') }}">Learn how to install it</a>, or ignore this warning to continue. <img id="tor-warning-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
```

Comment:
Nit: we're duplicating `id`s for `disable-js`. We should use a dedicated id for the tor-browser warning if we need to refer to the element by id.